examples/geo_schelling_points/geo_schelling_points/server.py

```diff
@@ -1,10 +1,10 @@
 import mesa
+from mesa.visualization.ModularVisualization import ModularServer
```

Comment:
No need for import. Should have used the new namespace in the code below with `mesa.visualization.ModularServer`.

Reply:
Right! Totally forgot that. I've updated all examples to use simple namespace from Mesa in a separate commit.

Perhaps a similar simple namespace should be implemented for Mesa-Geo too, to be consistent with Mesa.

Reply:
Yes, agreed for Mesa-Geo simple namespace.